src/constants/const.ts

```diff
@@ -0,0 +1,2 @@
+export const API_KEY = process.env.REACT_APP_API_KEY;
+export const API_URL = process.env.REACT_APP_API_URL;
```

Comment:
if in your folder only one file - always give name indeex.ts or index,jsx

Reply:
in folders such as : helpers, constantsm types (global golders) always need file with name index.ts where will be imports
![image](https://github.com/user-attachments/assets/46d8e4fb-9a2c-4012-bf89-9948d3de3aed)
![image](https://github.com/user-attachments/assets/b8d98e5e-72f1-4287-9d2a-4c94de71dffc)